.storybook/preview.ts

```diff
@@ -4,10 +4,36 @@ import '@fortawesome/fontawesome-free/css/v4-shims.css';
 import {Preview} from '@storybook/react';
 import 'formiojs/dist/formio.builder.css';
 import 'react-tooltip/dist/react-tooltip.css';
+import 'leaflet/dist/leaflet.css';
+import * as L from 'leaflet';
+// @ts-ignore
+import iconRetinaUrl from 'leaflet/dist/images/marker-icon-2x.png';
+// @ts-ignore
+import iconUrl from 'leaflet/dist/images/marker-icon.png';
+// @ts-ignore
+import shadowUrl from 'leaflet/dist/images/marker-shadow.png';
 
 import {BuilderContextDecorator, ModalDecorator} from './decorators';
 import {reactIntl} from './reactIntl';
 
+/**
+ * fix leaflet images import - https://github.com/Leaflet/Leaflet/issues/4968
+ * @return {Void}
+ */
+const fixIconUrls = () => {
+  // @ts-ignore
+  delete L.Icon.Default.prototype._getIconUrl;
+  // the call to `require` ensures that the static assets are bundled along (or, for small
+  // images, inlined as base64)
+  L.Icon.Default.mergeOptions({
+    iconRetinaUrl: iconRetinaUrl,
+    iconUrl: iconUrl,
+    shadowUrl: shadowUrl,
+  });
+};
```

Comment:
I remember having to do the same a couple years ago on a Vue project with leaflet as a dep., funny to see this is still something

Reply:
Yeah leaflet authors themselves do not want to fix anything in their code base, and I get it, since it's specific to how bundlers treat this stuff.

I'm not even sure things work here or if this should be shipped in the builder, since we're not displaying a pin anyway. I'll take another look.